features/whitehall.feature

```diff
@@ -113,8 +113,14 @@ Feature: Whitehall
     Then the elapsed time should be less than 2 seconds
 
   @normal
-  Scenario: Whitehall assets are served
+  Scenario: Whitehall assets are redirected to the asset host
     Given I am testing through the full stack
+    When I request "/government/uploads/system/uploads/attachment_data/file/618167/government_dietary_recommendations.pdf"
+    Then I should be redirected to the asset host
```

Comment:
Do we need to check the filename for the redirect URL matches the given one?

Reply:
Oh I guess that's covered by checking for the 200 response later on.

Reply:
Yeah, I think it's good enough to check the host given that the next scenario checks that we can successfully request the attachment. I considered removing the redirect test given that it's well tested within Whitehall but decided that it shouldn't do any harm to test it here too.